tardis/montecarlo/tests/test_rpacket.c

```diff
@@ -0,0 +1,16 @@
+#include "../src/rpacket.h"
+#include <assert.h>
+#include <stdlib.h>
+
+void test_rpacket_get_nu(){
+	rpacket_t * rp = (rpacket_t *) malloc(sizeof(rpacket_t));
+	double value = 10.2;
+	rpacket_set_nu(rp, value);
+	assert(value == rpacket_get_nu(rp));
```

Comment:
assert will end the process, so you wan't get good error messages or other tests running

raising a Python exception from C looks like this:
```c
PyErr_Format(PyExc_AssertionError, "value %g not %g", value, rpacket_get_nu(rp)))
// cleanup
return NULL;
```